profusion/android-enhanced-video-player
# Add JitPack configuration

## Description

Add jitpack and release configuration to the project



## Related Issues


- Closes #3 




### Pull request checklist




## How to test it

1. Go to https://jitpack.io/
2. Lookup for `profusion/android-enhanced-video-player`
3. In `branches` tab click in `Get it` on `feat~jitpack-config-SNAPSHOT` version
4. Wait for it to load or check `Builds` tab if the log is greened



## Visual reference

![image](https://github.com/profusion/android-enhanced-video-player/assets/9094028/9155a79a-0c31-4178-ae1d-38b7a8cfcc80)
